benchexec/baseexecutor.py

```diff
@@ -88,14 +88,21 @@ def _start_execution(
             and the result of parent_cleanup_fn (do not use os.wait)
         """
 
+        from_child, to_parent = os.pipe()
+
         def pre_subprocess():
+            os.close(from_child)
+
             # Do some other setup the caller wants.
             child_setup_fn()
 
             # put us into the cgroup(s)
             pid = os.getpid()
             cgroups.add_task(pid)
 
+            os.write(to_parent, str(pid).encode())
+            os.close(to_parent)
```

Comment:
If we decide to keep this, it would need to move above the `cgroups.add_task(pid)` call, because that starts the measurement.

But considering the different semantics to the container case, and the fact that we do not have a use case for it, I would tend to omit it.